.github/workflows/multi-repo-build.yml

```diff
@@ -7,16 +7,6 @@ on:
         description: 'Space-separated list of GitHub repos (e.g., "ocaml-dune/notty ocaml/ocaml-re")'
         required: true
         type: string
-      depext_linux:
```

Comment:
Getting rid of this isn't probably the best idea in the end. I think I will change it so that the default uses `dune show depext` but it can be overridden. There are some packages like https://opam.ocaml.org/packages/conf-libcurl/ that don't appear to correctly report depexts.

Reply:
Having an option to override the default depexts sounds good enough to me. It should cover most of the repos we want to test.